sdk/search/Azure.Search.Documents/src/Generated/Models/AutocompleteOptions.Serialization.cs

```diff
@@ -21,6 +21,10 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
                 writer.WritePropertyName("search");
                 writer.WriteStringValue(SearchText);
             }
+            else
+            {
+                writer.WriteNull("search");
```

Comment:
In most of the Search API, explicitly sending nulls isn't any different than omitting the property altogether (because we don't support PATCH). But for the Indexing API, it matters because for document merges, null means "delete this field value" while a missing property means "no change".

Reply:
Is that on any collection properties, though?

/cc @tg-msft since this impacts (hopefully not adversely) batching.

Reply:
Not sure I understand the question.

Reply:
It was to @brjohnstmsft - whether any of those particular APIs he mentioned where `null` is significant and destructive are impacted by this change.

Reply:
@heaths Good question. I misspoke earlier -- for collections, the service won't accept explicit `null` in the Index API. In that case, for `merge`, an empty JSON array means "delete the field value" and omitting the collection property entirely means "no change".

Reply:
@pakrym sounds like we'll need a change to not serialize nulls.

Reply:
I don't think there is a way to set null to the property so it doesn't really matter. There would be another change that cleans up the extra checks.

Reply:
> I don't think there is a way to set null to the property so it doesn't really matter.

I verified, at least with Search, that is indeed the case. Signing off.